### PR TITLE
sync: add page

### PR DESCRIPTION
**Pull request by sbrl**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/sync.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# sync
+
+> Synchronises all pending write operations to the appropriate disk.
```
Comment (agnivade) [inline pages/common/sync.md]: Maybe its better to use the word "flush". I know "synchronises" matches with the command, but filesystem and the buffer cache are not meant to be in sync at all. It all totally depends on the OS implementation.

File: pages/common/sync.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- Synchronise all pending write operations on all disks:
+
+`sync`
+
+- Synchronise all pending write operations on a single file to disk:
+
+`sync {{path/to/file.ext}}`
```
Comment (agnivade) [inline pages/common/sync.md]: I think its fine to remove the ".ext"

File: pages/common/sync.md (hunk 1 above)
Comment (sbrl) [reply pages/common/sync.md]: Yeah, I agree flush makes it clearer.

File: pages/common/sync.md (hunk 2 above)
Comment (sbrl) [reply pages/common/sync.md]: Sure - removed.

**Comment by sbrl** [comment]
Updated, @agnivade. Thanks for the suggestions!

**Comment by sbrl** [comment]
Merged due to inactivity :smiley_cat:

**Pull request merged**